driver-core/src/main/com/mongodb/MongoClientSettings.java

```diff
@@ -476,6 +481,18 @@ public Builder autoEncryptionSettings(final AutoEncryptionSettings autoEncryptio
             return this;
         }
 
+        // Package-private to provide interop with MongoClientOptions
```

Comment:
Previously these were not settable here, but they are in MongoClientOptions, so if we want to be able to transform options to settings then we need to make these properties.  Luckily it's in the same package so they don't have to be public.